cluster.yml

```diff
@@ -37,6 +37,11 @@
     - { role: kubernetes-apps/lib, tags: apps }
     - { role: kubernetes-apps/network_plugin, tags: network }
 
+- hosts: calico-rr
```

Comment:
nit: you could hide that as an optional dependency in the plugin's meta. Just to not make the cluster.yml to look more coplex